tools-local/tasks/mobile.tasks/CreateWasmBundle/DownloadTimeZoneData.cs

```diff
@@ -0,0 +1,103 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Net;
+using System.Reflection;
+using System.Diagnostics;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+public class DownloadTimeZoneData : Task
+{
+    [Required]
+    public string? InputDirectory { get; set; }
+
+    [Required]
+    public string? OutputDirectory { get; set; }
+
+    [Required]
+    public string? Version { get; set; }
+
+    private void DownloadTimeZoneDataSource() 
+    {
+        List<string> files = new List<string>() {"africa", "antarctica", "asia", "australasia", "etcetera", "europe", "northamerica", "southamerica", "zone1970.tab"};
+        using (var client = new WebClient())
+        {
+            Console.WriteLine("Downloading TimeZone data files");
```

Comment:
@tqiu8  Use `Log.LogMessageFromText ("Downloading TimeZone data files", MessageImportance.Low);` instead of CWL. This way, the message would end up in the logs correctly. Could you fix this as a follow up?